### PR TITLE
Call react from input.

### PR DESCRIPTION
**Pull request by BorislavkoDumic**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,10 +1,38 @@
...
+let myElement = document.getElementById("biologis-json-converter-helper");
+if (myElement === null) {
+  let newElement = document.createElement("div");
+  newElement.setAttribute("id", "biologis-json-converter-helper");
+  document.body.appendChild(newElement);
+}
+
+function changeInputText(inputEl, valuesString) {
+  inputEl.value = valuesString;
+  document.getElementById("biologis-json-converter-helper").style.display =
```
Comment (attrib) [inline src/index.js]: You could use `myElement` here now.
Also myElement is not a good name ;)

File: src/index.js [hunk 2]
```diff
@@ -1,10 +1,38 @@
...
+    "none";
+}
+let inputFields = document.getElementsByClassName("jsonHelper");
+for (let i = 0; i < inputFields.length; i++) {
+  inputFields[i].addEventListener("click", callRender);
+}
+
+function callRender(event) {
+  document.getElementById("biologis-json-converter-helper").style.display =
```
Comment (attrib) [inline src/index.js]: You could use `myElement` here now.

File: src/index.js [hunk 3]
```diff
@@ -1,10 +1,38 @@
...
+    "block";
+  ReactDOM.render(
+    <App
+      onChange={valuesString => changeInputText(event.target, valuesString)}
+    />,
+    document.getElementById("biologis-json-converter-helper")
```
Comment (attrib) [inline src/index.js]: You could use `myElement` here now.

File: src/reducers/overlay.js [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
-    case OPEN_OVERLAY:
-      return {
-        ...state,
-        showOverlay: true
-      };
+    // case OPEN_OVERLAY:
+    //   return {
+    //     ...state,
+    //     showOverlay: true
+    //   };
```
Comment (attrib) [inline src/reducers/overlay.js]: you could remove this now

File: src/reducers/inputFields.js [hunk 5]
```diff
@@ -5,7 +5,8 @@ import {
   CHANGE_VALUE,
   CHANGE_LABEL,
   SAVE_VALUES,
-  CANCEL
+  CANCEL,
+  GET_VALUES
```
Comment (attrib) [inline src/reducers/inputFields.js]: Not used yet?

File: src/App.js [hunk 6]
```diff
@@ -3,13 +3,18 @@ import Input from "./components_redux/Input";
...
 import reducer from "./reducers";
+import { openOverlay } from "./actions";
 
 const store = createStore(reducer);
 
-function App() {
+function App(props) {
+  if (props.testProp) {
+    store.dispatch(openOverlay());
+  }
```
Comment (attrib) [inline src/App.js]: not used anymore

File: src/index.js (hunk 1 above)
Comment (BorislavkoDumic) [reply src/index.js]: I will change the name, just wanted you to review first to see if it's okay.

File: src/index.js (hunk 2 above)
Comment (BorislavkoDumic) [reply src/index.js]: Okay, I will change that.

File: src/index.js (hunk 3 above)
Comment (BorislavkoDumic) [reply src/index.js]: Okay, I will change that.

File: src/reducers/overlay.js (hunk 4 above)
Comment (BorislavkoDumic) [reply src/reducers/overlay.js]: I will remove this.

File: src/App.js (hunk 6 above)
Comment (BorislavkoDumic) [reply src/App.js]: I will remove this.

**Comment by BorislavkoDumic** [comment]
> Dont forget to adapt the tests.

Okay, I will fix everything about tests.

File: src/reducers/overlay.js [hunk 7]
```diff
@@ -1,16 +1,9 @@
-import { OPEN_OVERLAY, SAVE_VALUES, CANCEL } from "../utils/actionTypes";
+import { SAVE_VALUES, CANCEL } from "../utils/actionTypes";
```
Comment (attrib) [inline src/reducers/overlay.js]: is anything still needed in here?

File: src/index.js [hunk 8]
```diff
@@ -4,36 +4,32 @@ import "./index.css";
...
 import * as serviceWorker from "./serviceWorker";
 
-let myElement = document.getElementById("biologis-json-converter-helper");
-if (myElement === null) {
-  let newElement = document.createElement("div");
-  newElement.setAttribute("id", "biologis-json-converter-helper");
-  document.body.appendChild(newElement);
+let formElement = document.getElementById("biologis-json-converter-helper");
+if (formElement === null) {
+  var newFormElement = document.createElement("div");
```
Comment (attrib) [inline src/index.js]: why creating a new variable here?
why using `var` not `let`?

**Pull request merged**